shell/renderer/api/electron_api_web_frame.cc

```diff
@@ -449,17 +449,13 @@ class WebFrameRenderer final : public gin::Wrappable<WebFrameRenderer>,
   }
 
   double GetZoomLevel(v8::Isolate* isolate) {
-    double result = 0.0;
     content::RenderFrame* render_frame;
     if (!MaybeGetRenderFrame(isolate, "getZoomLevel", &render_frame))
-      return result;
+      return 0.0f;
 
-    mojo::AssociatedRemote<mojom::ElectronWebContentsUtility>
-        web_contents_utility_remote;
-    render_frame->GetRemoteAssociatedInterfaces()->GetInterface(
-        &web_contents_utility_remote);
-    web_contents_utility_remote->DoGetZoomLevel(&result);
```

Comment:
Since we're no longer using `DoGetZoomLevel`, we can remove it from our mojo interface and all of the associated code in `/shell/browser`

https://github.com/electron/electron/blob/3dad07f3383615bd608a46f6b6682ffaffa667c1/shell/common/web_contents_utility.mojom#L18-L19